### ISSUE TITLE
Error: There were invalid base64 characters in the input text.

### ISSUE THREAD

**Issue by dylanjha** [danialfarid/ng-file-upload#1809]
> There were invalid base64 characters in the input text.
> Valid base64 characters are A-Z, a-z, 0-9, ' + ', ' / ',and "="\n'
> Expect errors in decoding.

This gets logged to the console, and this almost always leads to a corrupted file upload. Would it be better to treat this as an error instead of silently failing and treating it as a success?

**Comment by danialfarid** [comment]
How your code is look like? Where do you see this error?

**Comment by dylanjha** [comment]
I found the source of the error, it was setting resizeOptions to `image/png' format. When selecting a jpeg this led to the error.

My error has now been fixed, but more generally it feels like this situation should not be a silent failure, I would expect the error handling callback to be fired.

I'd be happy to build this change into the library. Thank you for maintaining, I <3 this uploader for angular
